packages/world_countries/README.md

```
@@ -48,6 +48,10 @@ To preview the demo from the example, you can visit [this web page](https://tsin
 
 > Note: Please keep in mind that demo was built with Flutter WASM which is in very early alpha stage, so performance might be affected.
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/tsinis/sealed_world?hide_repo_select=true&ref=main)
+<a href="https://idx.google.com/import?url=https%3A%2F%2Fgithub.com%2Ftsinis%2Fsealed_world%2Ftree%2Fmain%2Fpackages%2Fworld_countries%2Fexample">
```

Markdownlint (0.37.0)

Element: a
Inline HTML

(MD033, no-inline-html)